docker/mariadb/gk-waypointy-country.sql

```diff
@@ -0,0 +1,153 @@
+INSERT INTO `gk-waypointy-country` (`kraj`, `country`) VALUES
+('Ägypten',	'Egypt'),
+('Albania',	'Albania'),
+('Albanien',	'Albania'),
+('Åotwa',	'Latvia'),
+('Argentyna',	'Argentina'),
+('Armenia',	'Armenia'),
+('Australien',	'Australia'),
+('Austria',	'Australia'),
+('Belarus (Weißrußland)',	'Belarus'),
+('Belgia',	'Belgium'),
+('Belgien',	'Belgium'),
+('BiaÅ‚oruÅ›',	'Belarus'),
+('Białoruś',	'Belarus'),
+('Bosnien-Herzegowina',	'Bosnia and Herzegovina'),
+('Brasilien',	'Brazil'),
+('BuÅ‚garia',	'Bulgaria'),
+('Bulgarien',	'Bulgaria'),
+('Chiny',	'China'),
+('Chorwacja',	'Croatia'),
+('Costa Rica',	'Costa Rica'),
+('Cypr',	'Cyprus'),
+('Czeska Republika',	'Czech Republic'),
+('DÃ¤nemark',	'Denmark'),
+('Dänemark',	'Denmark'),
+('Dania',	'Denmark'),
+('Demokratische Volksrepublik Korea',	'North Korea'),
+('Deutschland',	'Germany'),
+('Dominikanische Republik',	'Dominican Republic'),
+('Ecuador',	'Ecuador'),
+('Egipt',	'Egypt'),
+('Estonia',	'Estonia'),
+('FÃ¤rÃ¶er (zu DÃ¤nemark)',	'Faroe Islands'),
+('Falklandinseln',	'Falkland Islands'),
+('Finlandia',	'Finland'),
+('Finnland',	'Finland'),
+('Francja',	'France'),
+('Frankreich',	'France'),
+('Georgien',	'Georgia'),
+('Gibraltar',	'Gibraltar'),
+('GrÃ¶nland',	'Greenland'),
+('Grecja',	'Greece'),
+('Griechenland',	'Greece'),
+('GroÃŸbritannien',	'United Kingdom'),
+('Grönland',	'Greenland'),
+('Großbritannien',	'United Kingdom'),
+('Hiszpania',	'Spain'),
+('Holandia',	'Netherlands'),
+('Indie',	'India'),
+('Indien',	'India'),
+('Irland',	'Ireland'),
+('Irlandia',	'Ireland'),
+('Island',	'Iceland'),
+('Islandia',	'Iceland'),
+('Israel',	'Israel'),
+('Italien',	'Italy'),
+('Jemen',	'Yemen'),
+('KambodÅ¼a',	'Cambodia'),
+('Kanada',	'Canada'),
+('Kapverden',	'Cape Verde'),
+('Kasachstan',	'Kazakhstan'),
+('Kenia',	'Kenya'),
+('Kirgistan',	'Kyrgyzstan'),
+('Kroatien',	'Croatia'),
+('Liechtenstein',	'Liechtenstein'),
+('Litwa',	'Lithuania'),
+('Luxemburg',	'Luxembourg'),
+('Malaysia',	'Malaysia'),
+('Malediven',	'Maldives'),
+('Malta',	'Malta'),
+('Marokko',	'Morocco'),
+('Maroko',	'Morocco'),
+('Mauritius',	'Mauritius'),
+('Mazedonien',	'Macedonia'),
+('MoÅ‚dawia',	'Moldova'),
+('Monako',	'Monaco'),
+('Montenegro',	'Montenegro'),
+('Namibia',	'Namibia'),
+('Nepal',	'Nepal'),
+('Neuseeland',	'New Zealand'),
+('Niderlandy',	'Netherlands'),
+('NiederlÃ¤ndische Antillen',	'Netherlands Antilles'),
+('Niederlande',	'Netherlands'),
+('Niederländische Antillen',	'Netherlands Antilles'),
+('Niemcy',	'Germany'),
+('Norwegen',	'Norway'),
+('Norwegia',	'Norway'),
+('Österreich',	'Austria'),
+('Polen',	'Poland'),
+('Polska',	'Poland'),
+('Portugal',	'Portugal'),
+('Portugalia',	'Portugal'),
+('Rosja',	'Russia'),
+('Ruanda',	'Rwanda'),
+('RumÃ¤nien',	'Romania'),
+('Rumänien',	'Romania'),
+('Rumunia',	'Romania'),
+('Russische FÃ¶deration',	'Russia'),
+('Russische Föderation',	'Russia'),
+('SÅ‚owacja',	'Slovakia'),
+('SÃ¼dgeorgien und die SÃ¼dlichen Sandwichinseln',	'South Georgia and the South Sandwich Islands'),
```

Comment:
Hum, there are many encoding problems in this file... In production database, this entry is `Südgeorgien und die Südlichen Sandwichinseln`